cico_setup.sh

```diff
@@ -11,7 +11,7 @@ set -e
 function load_jenkins_vars() {
   if [ -e "jenkins-env" ]; then
     cat jenkins-env \
-      | grep -E "(DEVSHIFT_TAG_LEN|DEVSHIFT_USERNAME|DEVSHIFT_PASSWORD|JENKINS_URL|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId)=" \
+      | grep -E "(IMG_REPO_TAG_LEN|IMG_REPO_USERNAME|IMG_REPO_PASSWORD|JENKINS_URL|GIT_BRANCH|GIT_COMMIT|BUILD_NUMBER|ghprbSourceBranch|ghprbActualCommit|BUILD_URL|ghprbPullId)=" \
```

Comment:
I don't know from where `DEVSHIFT_*` is coming. Might be env variable of Centos-CI. Can you make sure we are doing the right thing here?

Reply:
That is correct, `DEVSHIFT_USERNAME` and `DEVSHIFT_PASSWORD` are coming from CentOS-CI, however if migrating to quay, we should be using `QUAY_USERNAME` and `QUAY_PASSWORD` instead. Can you gobally replace `DEVSHIFT_USERNAME` and `DEVSHIFT_PASSWORD` with their `quay` counterparts? also, please keep `DEVSHIFT_TAG_LEN` as is, that's  a special variable that contains the expected length of git commit hash and can't be renamed.

Also, I recommend you to no longer use such a mechanismo to load the env, but rather use this:

```bash
function load_jenkins_vars() {
  if [ -e "jenkins-env.json" ]; then
    eval "$(./env-toolkit load -f jenkins-env.json \
        DEVSHIFT_TAG_LEN \
        QUAY_USERNAME \
        QUAY_PASSWORD \
        JENKINS_URL \
        GIT_BRANCH \
        GIT_COMMIT \
        BUILD_NUMBER \
        ghprbSourceBranch \
        ghprbActualCommit \
        BUILD_URL \
        ghprbPullId)"
  fi
}
```

It's a much safer approach as it's resilient to env vars with spaces, newlines and special chars.